source/macos/AdaptiveCards/AdaptiveCards/Views/ACRDateField.swift

```diff
@@ -261,3 +233,11 @@ extension NSPopover {
         super.draw(drawRect)
     }
 }
+
+extension ACRDateField: ACRTextFieldDelegate {
+    func acrTextFieldDidSelectClear(_ textField: ACRTextField) {
+        if textField == self.textField {
+            selectedDate = nil
```

Comment:
This is a good check! 
But since this can now only hold one `textField`, this check can be omitted.